src/command_modules/azure-cli-network/azure/cli/command_modules/network/_params.py

```diff
@@ -29,14 +63,24 @@
 virtual_network_name_type = CliArgumentType(options_list=('--vnet-name',), metavar='VNET_NAME', help='The virtual network (VNET) name.', completer=get_resource_name_completion_list('Microsoft.Network/virtualNetworks'))
 subnet_name_type = CliArgumentType(options_list=('--subnet-name',), metavar='SUBNET_NAME', help='The subnet name.')
 load_balancer_name_type = CliArgumentType(options_list=('--lb-name',), metavar='LB_NAME', help='The load balancer name.', completer=get_resource_name_completion_list('Microsoft.Network/loadBalancers'), id_part='name')
+private_ip_address_type = CliArgumentType(help='Static private IP address to use.', validator=validate_private_ip_address)
 
 choices_ip_allocation_method = [e.value.lower() for e in IPAllocationMethod]
+choices_private_ip_address_version = [e.value.lower() for e in privateIpAddressVersion]
```

Comment:
do we need this?  The choices should display on their own.  If we frequently need to change case, we might want to handle it more generically